compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -292,6 +319,7 @@ impl<'tcx> Body<'tcx> {
 
         let mut body = Body {
             phase: MirPhase::Built,
+            pass_count: 1,
```

Comment:
That looks like pretty fragile initialization. Is this always 1 initially?

Reply:
Yeah, this is a counter that is always set/reset to 1 everywhere.